brands/shop/newsagent.json

```
@@ -41,6 +41,36 @@
       "shop": "newsagent"
     }
   },
+  "shop/newsagent|Martin's": {
+    "count": 53,
```

This entry did not exist before, so it should be
```suggestion
    "nocount": true,
```